src/server/server.ts

```diff
@@ -20,43 +26,67 @@ const forceSSL = () => {
     };
 };
 
+
 // Instruct the app to use the forceSSL middleware
 if (process.env.NODE_ENV === 'production') {
     app.use(forceSSL());
     daoConfig.ssl = true;
 }
 
-const dao = new Dao(daoConfig);
 
-dao.connect().catch(error => {
-    console.log(error);
-    process.exit(1);
-});
+// Entity DAOs
+const usersDao = new UsersDao(daoConfig);
+const teamsDao = new TeamsDao(daoConfig);
+const singlesGamesDao = new SinglesGamesDao(daoConfig);
+const doublesGamesDao = new DoublesGamesDao(daoConfig);
+
 
 // Run the app by serving the static files in the dist directory
 app.use(express.static(path.join(__dirname + '/public')));
 
+
+// Parse JSON bodies
+app.use(bodyParser.urlencoded({ extended: true })) // parse application/x-www-form-urlencoded
+app.use(bodyParser.json())
+
+
 // Return all USERS
 app.get('/api/users', (req, res) => {
-    dao.getUsers(req, res);
+    usersDao.getUsers(req, res);
 });
 
 
 // Return all TEAMS
 app.get('/api/teams', (req, res) => {
-    dao.getTeams(req, res);
+    teamsDao.getTeams(req, res);
 });
 
 
 // Return all SINGLES GAMES
 app.get('/api/singles_games', (req, res) => {
-    dao.getSinglesGames(req, res);
+    singlesGamesDao.getSinglesGames(req, res);
 });
 
 
 // Return all DOUBLES GAMES
 app.get('/api/doubles_games', (req, res) => {
-    dao.getDoublesGames(req, res);
+    doublesGamesDao.getDoublesGames(req, res);
+});
+
+
+// Create new USER
+app.post('/api/create_user', (req, res) => {
+    let display_name = req.body.display_name;
```

Comment:
I think it would be more appropriate for this logic to be inside the UsersDao object and the router just forwards the request to it.

Reply:
Agreed - will make that change.